pkg/logger/event/event.go

```diff
@@ -17,39 +17,48 @@
 package event
 
 import (
-	public_source_ip "github.com/AccelByte/public-source-ip"
 	"github.com/emicklei/go-restful"
 	"github.com/fatih/structs"
 	"github.com/sirupsen/logrus"
 )
 
 const (
 	eventLogAttribute     = "EventLog"
-	eventType             = "event"
 	millisecondTimeFormat = "2006-01-02T15:04:05.999Z07:00"
+	traceIDKey            = "X-Ab-TraceID"
+	sessionIDKey          = "X-Ab-SessionID"
 )
 
 type event struct {
 	ID               int                    `structs:"event_id"`
+	Type             int                    `structs:"event_type"`
+	EventLevel       int                    `structs:"event_level"`
+	Resp             int                    `structs:"resp"`
+	Service          string                 `structs:"service"`
+	ClientIDs        []string               `structs:"client_ids"`
 	UserID           string                 `structs:"user_id"`
+	TargetUserIDs    []string               `structs:"target_user_ids"`
 	Namespace        string                 `structs:"namespace"`
-	ClientID         string                 `structs:"client_id"`
-	TargetUserID     string                 `structs:"target_user_id"`
 	TargetNamespace  string                 `structs:"target_namespace"`
+	TraceID          string                 `structs:"trace_id"`
+	SessionID        string                 `structs:"session_id"`
+	additionalFields map[string]interface{} `structs:"-"`
 	Realm            string                 `structs:"realm"`
-	SourceIP         string                 `structs:"source_ip"`
-	LogType          string                 `structs:"log_type"`
+	topic            string                 `structs:"-"`
+	Action           string                 `structs:"action"`
+	Status           string                 `structs:"status"`
 	level            logrus.Level           `structs:"-"`
-	message          []interface{}          `structs:"-"`
-	additionalFields map[string]interface{} `structs:"-"`
+	Privacy          bool                   `structs:"privacy"`
```

Comment:
we don't need this field and to be in the log

Reply:
I have updated it